test/signup.test.js

```diff
@@ -0,0 +1,127 @@
+const chai = require('chai');
+const app = require('../index');
+const request = require('supertest')(app);
```

Comment:
I have set up the test to use `chai-http`, can you check out the `index.test.js` file and follow the testing pattern. So that everyone's test will be consistent.